storage/cloud-client/snippets.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to perform basic operations on blobs
+(objects) in a Google Cloud Storage bucket.
```

Comment:
Fair, but I can see the argument that you're not really expected to describe what it does in one line, just give a teaser.

For example, you can kill "this application" (implied by context) and write "Demonstrates how to perform basic operations on blobs." Then the rest of the summary describes what the basic operations are, what the summary is, etc. 

On the one hand I agree with you because I've felt the same way, on the other hand I'm not a big fan of us breaking one of the conventions that _both_ Google and PEP agree on. /shruggie